decidim-blogs/spec/helpers/decidim/blogs/admin/post_helper_spec.rb

```diff
@@ -4,6 +4,8 @@
 
 module Decidim::Blogs::Admin
   describe PostsHelper, type: :helper do
+    include Decidim::DisabledRedesignLayout
```

Comment:
I don't think we should disable the redesigned layout here, right?

Or why do you want to do that?

Reply:
OK makes sense, thanks for the explanation!

And since it's only for the admin panel test, I guess that's fine anyways for the time being.